kirega/iReporter
# #162366425 ft database connection

#### What does this PR do?
Integrates postgres into the system.
#### Description of Task to be completed?
Install psycopg2
Create the connections
Write the scripts for creating the tables.
Ensure the scripts run at runtime 
#### How should this be manually tested?

- Create a new database on postgres database called **ireporter** using `psql`
- After cloning the repo
- Edit the `app/db_con` file with your credentials.
- Now following the readme.md instructions setup and run the flask app 
- Using `psql ireporter` run `SELECT * from public."User"`.
- You should see an empty table was created with the correct columns
#### What are the relevant pivotal tracker stories?
[#162366425](https://www.pivotaltracker.com/story/show/162366425)
